WooCommerce/Classes/ViewRelated/Feature Highlight/FeatureHighlightStore.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+struct FeatureHighlightStore {
+    private let userStore: UserDefaults
+
+    init(userStore: UserDefaults = UserDefaults.standard) {
+        self.userStore = userStore
+    }
+
+    var didDismissTooltip: Bool {
+        get {
+            return userStore.bool(forKey: UserDefaults.Key.didUserDismissTooltip.rawValue)
+        }
+        set {
+            userStore.set(newValue, forKey: UserDefaults.Key.didUserDismissTooltip.rawValue)
+        }
+    }
+
+    var writeWithAITooltipCounter: Int {
```

Comment:
nit: maybe the name can be more descriptive
```suggestion
    var numberOfTimesTooltipIsShown: Int {
```

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/b18551264d2497f159b4eb0dab4df66575a7d302